package.json

```diff
@@ -27,14 +29,23 @@
   },
   "devDependencies": {
     "@babel/core": "^7.14.3",
+    "@babel/preset-env": "^7.14.5",
+    "@babel/preset-react": "^7.14.5",
+    "@types/enzyme": "^3.10.8",
+    "@types/jest": "^26.0.23",
     "@types/plotly.js": "^1.54.10",
     "@types/react": "^17.0.9",
     "@types/react-dom": "^17.0.6",
     "@types/react-router-dom": "^5.1.7",
     "@typescript-eslint/eslint-plugin": "^4.26.1",
     "@typescript-eslint/parser": "^4.26.1",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.2",
+    "enzyme": "^3.11.0",
     "eslint": "^7.28.0",
+    "jest": "^27.0.4",
+    "jest-canvas-mock": "^2.3.1",
```

Comment:
Is this library required?

Reply:
Jest is used. jest-canvas-mock is also used as we have plotly.js in the dashboard, which requires this lib to setup a canvas mock for testing. See `jest.config.js` for the setup